src/renderer/assets/blocks/block-svg-helper.tsx

```diff
@@ -0,0 +1,16 @@
+import { useTheme } from "@/renderer/providers/themeProvider";
+
+const withThemeSVG = (
+  svgMap: Record<string, (theme?: "dark" | "light") => React.JSX.Element>,
```

Comment:
Hey Syed, could you please make sure not type the key of a Record as `string`? Since in this case we know what are all the possible keys right

Reply:
@itsjoeoui I believe using a string type for the key here is appropriate, as the code always checks if the block name is in the map object before loading the value. So, even if you register something unusual in the map object, it won't be selected. And, to be honest, the key is actually the block name, and there are so many blocks that mentioning their names individually would be cumbersome.